geode-core/src/distributedTest/java/org/apache/geode/cache/client/internal/SSLDualServerNoClientAuthDUnitTest.java

```diff
@@ -0,0 +1,213 @@
+        createTempFileFromResource(SSLDualServerNoClientAuthDUnitTest.class, SERVER_1_TRUSTSTORE)
+            .getAbsolutePath();
+    gemFireProps.setProperty(SSL_ENABLED_COMPONENTS, "cluster");
+    gemFireProps.setProperty(SSL_REQUIRE_AUTHENTICATION, "" + cacheServerSslRequireAuth);
+    gemFireProps.setProperty(SSL_KEYSTORE, "" + keyStore);
+    gemFireProps.setProperty(SSL_KEYSTORE_PASSWORD, "password");
+    gemFireProps.setProperty(SSL_TRUSTSTORE, "" + trustStore);
+    gemFireProps.setProperty(SSL_TRUSTSTORE_PASSWORD, "password");
+
+    StringWriter sw = new StringWriter();
+    PrintWriter writer = new PrintWriter(sw);
+    gemFireProps.list(writer);
+
+    Locator.startLocatorAndDS(0, new File(""), gemFireProps);
+
+    return Locator.getLocator().getPort();
+  }
+
+  private void setUpAndConnectToDistributedSystem(Integer locatorPort) throws Exception {
+    Properties gemFireProps = new Properties();
+
+    String cacheServerSslprotocols = "any";
+    String cacheServerSslciphers = "any";
+    boolean cacheServerSslRequireAuth = false;
+
+    System.setProperty("javax.net.debug", "all");
+    String keyStore;
+    String trustStore;
+    if (VM.getCurrentVMNum() == 1) {
+      keyStore =
+          createTempFileFromResource(SSLDualServerNoClientAuthDUnitTest.class, SERVER_1_KEYSTORE)
+              .getAbsolutePath();
+      trustStore =
+          createTempFileFromResource(SSLDualServerNoClientAuthDUnitTest.class,
+              SERVER_1_TRUSTSTORE)
+                  .getAbsolutePath();
+    } else {
+      keyStore =
+          createTempFileFromResource(SSLDualServerNoClientAuthDUnitTest.class, SERVER_2_KEYSTORE)
+              .getAbsolutePath();
+      trustStore =
+          createTempFileFromResource(SSLDualServerNoClientAuthDUnitTest.class,
+              SERVER_2_TRUSTSTORE)
+                  .getAbsolutePath();
+    }
+    gemFireProps.setProperty(SSL_ENABLED_COMPONENTS, "cluster");
+    gemFireProps.setProperty(SSL_REQUIRE_AUTHENTICATION, "" + cacheServerSslRequireAuth);
+    gemFireProps.setProperty(SSL_KEYSTORE, "" + keyStore);
+    gemFireProps.setProperty(SSL_KEYSTORE_PASSWORD, "password");
+    gemFireProps.setProperty(SSL_TRUSTSTORE, "" + trustStore);
+    gemFireProps.setProperty(SSL_TRUSTSTORE_PASSWORD, "password");
+
+    gemFireProps.setProperty(LOCATORS, "localhost[" + locatorPort + "]");
+
+
+    StringWriter sw = new StringWriter();
+    PrintWriter writer = new PrintWriter(sw);
+    gemFireProps.list(writer);
+    createCache(gemFireProps);
+
+    RegionFactory factory = cache.createRegionFactory(RegionShortcut.REPLICATE);
+    Region r = factory.create("serverRegion");
+    r.put("serverkey", "servervalue");
+  }
+
+  private void doServerRegionTest() {
+    Region<String, String> region = cache.getRegion("serverRegion");
+    assertEquals("servervalue", region.get("serverkey"));
+  }
+
+  private static Integer setUpLocatorTask() throws Exception {
+    return instance.setUpLocator();
```

Comment:
Do single line helper methods buy us much?  Same comment on the next few methods.